book/src/exec.md

```diff
@@ -0,0 +1,154 @@
+# Taiga execution model
+
+## On a high level
+Intents are a mechanism that allows users to enact a multiparty state transition. High level description of the flow:
+1. **Create**: A user creates an intent publishing the information of what they have and what they want
+2. **Gossip**: The intent goes to the intent gossip network and is gossiped around the solver nodes
+3. **Solve**: Solvers find matching intents and update the intent with partial transactions until the intent is fully satisfied
+4. **Finalize**: When the intent is satisfied, a transaction is created and published on the blockchain
+
+![img.png](img/exec_high.png)
```

Comment:
<img width="1158" alt="Screenshot 2022-10-24 at 11 54 00" src="https://user-images.githubusercontent.com/2612856/197510394-1057701d-12d9-4f9a-b934-54fa38b2e25d.png">

This is how I understand the high level execution model:
In circles we have the parties, arrows represent what is being sent, dotted lines represent actions.

I think it would be helpful to indicate that solvers and users are constantly listening to the gossip network.

From this and your diagram, users don't send intents directly to a solver. Is that correct?

Reply:
I focused on the fact that users send intents and solvers receive intents, and everything in between is out of scope, but maybe it is wrong. Also, I'm not sure that what happens in between is Taiga-specific, but if it is, it definitely should appear somewhere